nls/es/strings.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2013 Adobe Systems Incorporated. All rights reserved.
```

Comment:
I don't know about the copyright, you're free to change it or do you want me to correct it?